pkgs/applications/video/srt-live-server/default.nix

```diff
@@ -19,6 +20,12 @@ stdenv.mkDerivation rec {
   patches = [
     # https://github.com/Edward-Wu/srt-live-server/pull/94
     ./fix-insecure-printfs.patch
+
+    # https://github.com/Edward-Wu/srt-live-server/pull/127  # adds `#include <ctime>`
+    (fetchpatch {
+       url = "https://github.com/Edward-Wu/srt-live-server/commit/a7460c334735f54f4ad05ab302e0c3484eeb376b.patch";
+       sha256 = "sha256-UoNbyNf2IVXi97E9D4OIoPXmXUcS2e93nu7kDECYe0I=";
+    })
```

Comment:
Could you please just put the patch file in nixpkgs, so that we don't lose it when the PR is updated?

Reply:
Makes sense.  Done